client/src/components/advancedSelectWidget/AdvancedSelectOverlayRow.js

```diff
@@ -136,7 +136,7 @@ export const ReportDetailedOverlayRow = item => (
       <LinkTo modelType="Report" model={item} isLink={false} />
     </td>
     <td>
-      <span>{item.author ? item.author.name : "Unknown"}</span>
+      <span>{item.authors ? item.authors.join(", ") : "Unknown"}</span>
```

Comment:
`item.authors` is an array of objects. As a result authors gets displayed as `[Object object]`. Following change fixes this.
```
      <span>
        {item.authors
          ? item.authors.map(a => (
            <div key={a.uuid} style={{ whiteSpace: "nowrap" }}>
              {a.name}
            </div>
          ))
          : "Unknown"}
      </span>
```